send_sms.rb

```diff
@@ -0,0 +1,47 @@
+# Download the twilio-ruby library from twilio.com/docs/libraries/ruby
+# require 'twilio-ruby'
+#
+# class Menu
+#   attr_reader :account_sid, :auth_token,
+#               :client, :from, :to
+#
+#   def initialize(client = Twilio::REST::Client.new(account_sid, auth_token))
+#     @account_sid = 'ACbad1cc1fe45037b01a7fd207fb7f270c'
+#     @auth_token = '2223f6422dca48314885d82d94b3d005'
+#     @client = client
+#     @from = '+447480485499' # Your Twilio number
+#     @to = '+447828166080' # Your mobile phone number
+#   end
+#
+#   def send_message
+#     client.messages.create(
+#     from: from,
+#     to: to,
+#     body: "you've placed a takeaway order, it will arrive in 50mins"
+#     )
+#   end
+# end
+
+require 'twilio-ruby'
+
+account_sid = 'ACbad1cc1fe45037b01a7fd207fb7f270c'
```

Comment:
Use environment variables to store this data locally. Please remove it from GitHub (account SID, auth token and phone numbers) and use environment variables instead.